libraries/rush-sdk/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@rushstack/rush-sdk",
+  "version": "5.56.0",
+  "description": "An API for interacting with the Rush engine",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/rushstack.git",
+    "directory": "apps/rush-sdk"
+  },
+  "engines": {
+    "node": ">=5.6.0"
+  },
+  "engineStrict": true,
```

Comment:
Are these fields still relevant?

Reply:
Removed --it's only relevant for end user tools